source/collections/FloatVectorList.ooc

```diff
@@ -30,6 +35,19 @@ FloatVector: class extends VectorList<Float> {
 		result count = this count
 		result
 	}
+
+	discreteFourierTransform: static func (input: This) -> This {
```

Comment:
Also, where is `inverseDiscreteFourierTransform`, `fastFourierTransform`, and `inverseFastFourierTransform`?